tika-core/src/main/java/org/apache/tika/extractor/DefaultEmbeddedStreamTranslator.java

```diff
@@ -42,11 +47,23 @@
     }
 
     public DefaultEmbeddedStreamTranslator() {
-        this(getDefaultFilters(new ServiceLoader()));
+        this(new ServiceLoader());
+    }
+    
+    public DefaultEmbeddedStreamTranslator(boolean loadDynamicServices) {
+        this(new ServiceLoader(loadDynamicServices));
+        this.loadDynamicServices = loadDynamicServices;
+    }
+    
+    private DefaultEmbeddedStreamTranslator(ServiceLoader loader) {
+        this(getDefaultFilters(loader));
+        dynamicTranslators = loader.loadDynamicServiceProviders(EmbeddedStreamTranslator.class);
```

Comment:
I think that in case of 'loadDynamicServices' the 'dynamic translators' end up two times in the resulting list.